website/content/docs/plugins/hcp-support.mdx

```diff
@@ -0,0 +1,114 @@
+---
+description: |
+  The HCP Packer support allows plugins to be used with HCP and push metadata to the HCP Packer registry.
+page_title: Integration Program
+---
+
+# HCP Packer Support
+
+~> **Note:** HCP Packer is under active development, and we suggest plugin maintainers to keep up with the SDK changes
+for more on HCP Packer support.
+
+~> **Note:** If you're looking to develop a new plugin, please refer first to the [developing
+plugins](/docs/plugins/creation#developing-plugins) page.
+
+The HCP Packer support consists on making the Image metadata available to Packer through the builder artifact. Before pushing
+metadata to the HCP Packer registry, Packer will query for the Image metadata by looking for a [Image](https://github.com/hashicorp/packer-plugin-sdk/blob/b688c9db722f8e9685310b03ebcc55bfa6b7a674/packer/registry/image/image.go#L22)
+under a [key](https://github.com/hashicorp/packer-plugin-sdk/blob/b688c9db722f8e9685310b03ebcc55bfa6b7a674/packer/registry/image/image.go#L15)
+in the artifact.
+
+## Builder Artifact
+
+To make the plugin support HCP Packer, changes must be made to the build artifact. The artifact should return the appropriate
+Image metadata in `State(name string) interface{}` when the `name` is the `par.artifact.metadata` specific key.
+
+The packer-plugin-sdk implements the Image struct to make HCP Packer support easier.
+We provide the [`FromArtifact`](https://github.com/hashicorp/packer-plugin-sdk/blob/efe3f1229c88b517e236b06e927b0dba7dd9394e/packer/registry/image/image.go#L96) function
+to easily create an Image with the basic information contained in the Artifact. If customization is necessary, the `FromArtifact` accepts
+override functions as the second argument, and that can be used to set different values for the metadata.
+
+The [Image metadata](https://github.com/hashicorp/packer-plugin-sdk/blob/b688c9db722f8e9685310b03ebcc55bfa6b7a674/packer/registry/image/image.go#L22) structure is:
```

Comment:
I would suggest adding a link a link to https://pkg.go.dev/github.com/hashicorp/packer-plugin-sdk/packer/registry/image#Image
```suggestion
The [Image metadata](https://pkg.go.dev/github.com/hashicorp/packer-plugin-sdk/packer/registry/image#Image) structure is:
```